CocoaUPnP/Events/UPPEventServer.m

```diff
@@ -2,7 +2,7 @@
 // Copyright 2015 Arcam. See LICENSE file.
 
 #import "UPPEventServer.h"
-#import "GCDWebServerDataRequest.h"
+#import <GCDWebServers/GCDWebServers.h>
```

Comment:
I checked, and you're absolutely right. This will mean that we can no longer target iOS, as we enforce dynamic libraries. With iOS 10 being a few months away I think this is a reasonable step.